src/index.html

```diff
@@ -17,16 +17,38 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__container">
+      <form
+        action="/"
+        method="get"
+        class="search-bar"
+        data-qa="big"
+      >
+        <div class="icon">
+          <input
+            type="text"
+            data-qa="keypress"
+            placeholder="Try “Los Angeles“"
+            class="search__input big"
+          />
```

Comment:
Remember about correct indentation between parent and child elements. The <input> element inside the <div> should be indented by 2 spaces.